src/components/settings/data/hooks.js

```diff
@@ -47,6 +47,7 @@ export const useLinkManagement = (enterpriseUUID) => {
     };
   }, [enterpriseUUID]);
 
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   useEffect(loadLinks, []);
```

Comment:
We should generally avoid ignoring ESLint rules about `react-hooks/exhaustive-deps`. Is there a solution here to avoid needing to disable this rule?

Reply:
That would be to either add loadLinks to the dependency array or remove the array entirely. I have seen these being ignored for useEffect in other areas/repos as well. One such example would be in SSOConfigConfiguredCard.jsx